packages/ui-components/src/InlineAlert/InlineAlert.tsx

```diff
@@ -2,14 +2,12 @@ import React, { useMemo } from "react";
 import classNames from "classnames/bind";
 
 import styles from "./InlineAlert.module.scss";
-import { Icon, IconTint } from "../Icon/Icon";
-
-export type InlineAlertIntent = "info" | "error" | "success" | "warning";
+import { Icon, IconIntent } from "../Icon/Icon";
 
 const cx = classNames.bind(styles);
 
 interface InlineAlertBaseProps {
-  intent?: InlineAlertIntent;
+  intent?: IconIntent;
```

Comment:
I don't mind this here for now, but this makes me think long term we may need a type defined for Intents that are not component specific. That would leveraged by components like you've done here.

Reply:
Yes! I can make an issue for it in Github for ui-components.